src/_locales/es/messages.json

```diff
@@ -23,6 +23,9 @@
     "hide_twitter_blue": {
         "message": "Ocultar íconos de Twitter Blue"
     },
+    "revoke_verified": {
+        "message": "Revocar las insignias verificadas de los suscriptores de Twitter Blue"
```

Comment:
In spanish could be: "Quitar ícono de verificado a usuarios con Twitter Blue."

Reply:
Hi @luiserdef! Thank you for the suggestion! I made the change here and updated the english version as well for consistency.